source/postcard-rpc/src/server/impls/embassy_usb_v0_4.rs

```diff
@@ -81,6 +81,12 @@ pub mod dispatch_impl {
         pub cell: StaticCell<Mutex<M, EUsbWireTxInner<D>>>,
     }
 
+    /// A helper type for `static` storage of buffers and driver components
```

Comment:
So this is `WireStorage` without the USB state. The impl is copy&pasted and then I removed some lines.

A lot can be duplicated here, also in the original implementation. Ideally they'd all share snippets. It took me some time to get an idea of what's postcard specific and what not. Also, the poststation variant has only some small differences, which are IMO hard to spot.

Reply:
> Ideally they'd all share snippets

Or maybe the no-usb-state version could be a field of the original one.